### DISCUSSION TITLE
Recommended usage to stay up to date?

### DISCUSSION THREAD

**Discussion by onx2** [Sixze/ALS-Refactored#494] [Q&A] [answered]
How are people using this? Should I fork it or download zip then just check every once in a while to copy/paste code over?

**Comment by Rusty01** [comment, marked as answer]
The best thing to do is to NOT over-write any files with personal code. So when you add ALS to your project, just hook into it, or copy out the files you need to modify. That way if you do want to update ALS, you're not over-writing any of your work.  This is the method that I use and find it works well. 
Hope this helps.

**Reply by onx2** [reply]
Got it - thanks for the confirmation!

**Comment by ameaninglessname** [comment]
https://github.com/Sixze/ALS-Refactored/discussions/487#discussioncomment-9253531

**Comment by Magic-Summer** [comment]
Hi Jeff
This is basically my approach: 
1. Git pull regularly (maybe once a month) and note what changed.
2. If any blueprints you have copied from the ALS-Refactored plugin have changed, then you will need to investigate this further and possibly recreate them or some of the overridden functions inside them.
3. If assets have been renamed, you may need to delete the ALS-Refactored plugin `.\Plugins\ALS-Refactored\Binaries` folder and build again.
4. If you use [core redirects](https://github.com/Sixze/ALS-Refactored/discussions/395#discussioncomment-7428429) to changed assets, then you'll need to check these still work.

Good luck!
